practices/coordination.md

```diff
@@ -12,48 +12,86 @@ Here are the goals we optimize for in organizing our team coordination practices
 
 We want to do this with high efficiency so that people can quickly get on the same page and focus efforts around decision-making and collaboration. We also wish to do it in a 💯 remote-friendly way since we are split across many different time-zones.
 
+## Team development workflow
 
-## Projects and deliverables are our main units of work
+This section describes how our development team carries out its planning and day-to-day work.
+Here's a quick summary:
 
-A new project should be created for anything with a specific scope and set of deliverables that spans a reasonably long amount of time. 2i2c projects should all feed into the [2i2c organizational strategy and goals](../about/strategy.md).
+* [Deliverables](coordination:active-backlog) are discrete units of value that we bring to others. To be ready for work, they should be scoped properly, with enough information to complete them (e.g., a set of [tasks](coordination:tasks)).
```

Comment:
I think the link (coordination:active-backlog) is incorrect here...

Reply:
Ah so in MyST Markdown, when being used with Sphinx, the markdown links behave like `:any:` roles in Sphinx. So this means that you can include either:

- A relative path to another file (e.g., `../index.md`)
- A URL (e.g. `https://google.com`)
- A cross-reference (e.g., `mylabel`)

In this case, I'm referencing the `coordination:active-backlog` section which is defined here:

https://github.com/2i2c-org/team-compass/pull/73/files#diff-b0c9a716af7c8f8c76b0af475a8b92ce614b96b340678951c855a7febad7654dR62

Reply:
Yep, I quickly noted you wanted to cross-reference some other section... but my comment was triggered because I think the `coordination:active-backlog` reference is incorrect. Don't you want to reference the  `coordination: deliverables` section?

Reply:
ahhhh yep you are correct! I looked for a typo but hadn't considered that I'd just totally referenced the wrong section :-) thanks for spotting

Reply:
I think you still did not fix this one in your latest commits, right?

Reply:
weird - you are correct! I just committed an update now